patches/chrome-android-java-res-menu-main_menu.xml.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/android/java/res/menu/main_menu.xml b/chrome/android/java/res/menu/main_menu.xml
+index 4580a99b4e573076e5668eb71745ba18332e17a8..56ca462271d2d6535d2a04f66bcadb8d087ec85d 100644
+--- a/chrome/android/java/res/menu/main_menu.xml
++++ b/chrome/android/java/res/menu/main_menu.xml
+@@ -80,6 +80,8 @@
+             android:title="@string/menu_help" />
+         <item android:id="@+id/enter_vr_id"
+             android:title="@string/enter_vr" />
++        <item android:id="@+id/exit_id"
```

Comment:
Trying to avoid this patch.

Reply:
I think we can add our items dynamically by `Menu::add()`, but can't insert at specific position because upstream file doesn't specify `orderInCategory`
So, I think modifying `main_menu.xml` directly is the only way for adding our items at specific position. WDYT? @SergeyZhukovsky @bridiver

Reply:
Yeah, I didn't find any ways unfortunately as well

Reply:
At least you can make it one string instead of two to avoid more insertions

Reply:
Done.